content/en/blog/_posts/2021-12-16-kubernetes-is-moving-on-from-dockershim.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: blog
+title: "Kubernetes is moving on from Dockershim: commitments and next steps"
+date: 2021-12-16
```

Comment:
The date I'd use is 2021-12-20 - please update this (and adjust the path). That keeps the announcement separate from v1.23 post-release blogs.

Reply:
See https://github.com/kubernetes/website/pull/30972#discussion_r770804291 for an update.